Neos.Flow/Classes/Http/Request.php

```diff
@@ -590,27 +585,48 @@ function (&$value) {
     }
 
     /**
-     * Returns and array of all possibles "field paths" for the given array.
+     * Returns and array of all possible "field paths" for the given array.
      *
      * @param array $structure The array to walk through
      * @param string $firstLevelFieldName
      * @return array An array of paths (as strings) in the format "key1/key2/key3" ...
      */
-    protected function calculateFieldPaths(array $structure, $firstLevelFieldName = null)
+    protected function calculateFieldPaths(array $structure, string $firstLevelFieldName = null)
```

Comment:
This method is no longer needed, is it?
Otherwise we should at least add a `@deprecated` annotations IMO